Code/Tools/ProjectManager/Source/ProjectGemCatalogScreen.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <ProjectGemCatalogScreen.h>
+#include <PythonBindingsInterface.h>
+#include <GemCatalog/GemCatalogHeaderWidget.h>
+#include <GemCatalog/GemDependenciesDialog.h>
+#include <GemCatalog/GemFilterWidget.h>
+#include <GemCatalog/GemModel.h>
+#include <GemCatalog/GemRequirementDialog.h>
+
+#include <QDir>
+#include <QMessageBox>
+#include <QTimer>
+
+namespace O3DE::ProjectManager
+{
+    ProjectGemCatalogScreen::ProjectGemCatalogScreen(QWidget* parent)
+        : GemCatalogScreen(/*readOnly = */ false, parent)
+    {
+
+    }
+
+    ProjectManagerScreen ProjectGemCatalogScreen::GetScreenEnum()
+    {
+        return ProjectManagerScreen::ProjectGemCatalog;
+    }
+
+    ProjectGemCatalogScreen::EnableDisableGemsResult ProjectGemCatalogScreen::EnableDisableGemsForProject(const QString& projectPath)
```

Comment:
Terminology wise, while "enable" works, "disable" is incorrect.
The gem might still be active for the project just not explicitly.
More accurately this function is adding gems to explicitly activate and removing gems to explicit activate